src/L2ERC20.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import {ERC20} from "openzeppelin/token/ERC20/ERC20.sol";
+import {ERC20Votes} from "openzeppelin/token/ERC20/extensions/ERC20Votes.sol";
+import {ERC20Permit} from "openzeppelin/token/ERC20/extensions/ERC20Permit.sol";
+import {IWormhole} from "wormhole/interfaces/IWormhole.sol";
+
+contract L2ERC20 is ERC20Votes {
+  /// @notice The core bridge used to verify messages.
+  IWormhole coreBridge;
+
+  /// @notice The mapping of Wormhole chain id to cross chain contract address.
+  mapping(uint16 => bytes32) _applicationContracts;
+
+  /// @notice The mapping of a Wormhole message hash to boolean indicating whether the message was
+  /// completed or not.
+  mapping(bytes32 => bool) _completedMessages;
+
+  // @notice The address of the owner of the contract.
+  address owner;
```

Comment:
Wonder if we should just use OZ Ownable (which comes with the ability to change the owner too) since we're already relying on OZ for ERC20Votes as it is.

Reply:
Yes, good idea. I will update this

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/11/commits/833de90d174e2f1ef9d0cf0757e6398ad1085184